src/Database/Statement/PDOStatement.php

```diff
@@ -88,17 +88,14 @@ public function bindValue($column, $value, $type = 'string')
      * @return array|false Result array containing columns and values or false if no results
      * are left
      */
-    public function fetch($type = 'num')
+    public function fetch($type = parent::FETCH_TYPE_NUM)
     {
-        if ($type === 'num') {
+        if ($type === static::FETCH_TYPE_NUM) {
             return $this->_statement->fetch(PDO::FETCH_NUM);
         }
-        if ($type === 'assoc') {
+        if ($type === static::FETCH_TYPE_ASSOC) {
             return $this->_statement->fetch(PDO::FETCH_ASSOC);
         }
-        if ($type === 'obj') {
-            return $this->_statement->fetch(PDO::FETCH_OBJ);
-        }
```

Comment:
Won't removing this case break userland code that is currently passing 'obj'?

Reply:
I think having a constant for the `obj` type is ok and good. I previously only meant to recommend removing the obj methods.

Reply:
I see what you are saying. I'll put the constant back with the conditional for fetching objects.

Should I update fetch/fetchall doc blocks showing usage with constants instead of strings or leave it as is?